hugo/content/prometheus/_index.md

```diff
@@ -84,48 +85,51 @@ The following pgMonitor configuration files should be placed according to the fo
 
 If you need to modify them, see the notes in the files for more details and recommendations:
 
-- `/etc/systemd/system/prometheus.service.d/crunchy-prometheus-service-el7.conf`
-- `/etc/systemd/system/alertmanager.service.d/crunchy-alertmanager-service-el7.conf`
+- {{< shell >}}/etc/systemd/system/prometheus.service.d/crunchy-prometheus-service-rhel.conf{{< /shell >}}
+- {{< shell >}}/etc/systemd/system/alertmanager.service.d/crunchy-alertmanager-service-rhel.conf{{< /shell >}}
 
 The below files contain startup properties for Prometheues and Alertmanager.  Please review and modify these files as you see fit:
 
-- `/etc/sysconfig/prometheus`
-- `/etc/sysconfig/alertmanager`
+- {{< shell >}}/etc/sysconfig/prometheus{{< /shell >}}
+- {{< shell >}}/etc/sysconfig/alertmanager{{< /shell >}}
 
-The below files dictate how Prometheus and Alertmanager will behave at runtime for the purposes of using pgmonitor.  Please review each file below and follow the instructions in order to set things up:
+The below files dictate how Prometheus and Alertmanager will behave at runtime for the purposes of using pgMonitor.  Please review each file below and follow the instructions in order to set things up:
 
 | File                                     | Instructions |
 |------------------------------------------|--------------|
-| `/etc/prometheus/crunchy-prometheus.yml` | Modify to set scrape interval if different from the default of 30s. Activate alert rules and Alertmanager by uncommenting lines when set as needed. Service file provided by pgmonitor expects config file to be named `crunchy-prometheus.yml` |
-| `/etc/prometheus/crunchy-alertmanager.yml` | Setup alert target (e.g., SMTP, SMS, etc.), receiver and route information. Service file provided by pgmonitor expects config file to be named `crunchy-alertmanager.yml` |
-| `/etc/prometheus/alert-ruled.d/crunchy-alert-rules-\*.yml.example` | Update rules as needed and remove `.example` suffix. Prometheus config provided by pgmonitor expects `.yml` files to be located in `/etc/prometheus/alert-rules.d/` |
-| `/etc/prometheus/auto.d/*.yml` | You will need at least one file with a final `.yml` extension. Copy the example files to create as many additional targets as needed.  Ensure the configuration files you want to use do not end in `.yml.example` but only with `.yml`. Note that in order to use the provided Grafana dashboards, the extra "exp_type" label must be applied to the target and be set appropriately (pg or node). See the example target files provided for how to set the labels for postgres or node exporter targets. |
+| /etc/prometheus/crunchy-prometheus.yml | Modify to set scrape interval if different from the default of 30s. Activate alert rules and Alertmanager by uncommenting lines when set as needed. Activate blackbox_exporter monitoring if desired. Service file provided by pgMonitor expects config file to be named `crunchy-prometheus.yml` |
```

Comment:
Do the backticks in this (and others in this table) need to be converted to `shell` shortcodes, or they are fine in the table context?

Reply:
Our rules to not use decoration inside decoration, this mean no backticks inside table (table is decoration).

Reply:
Yeah I'd just removed the backticks all together here for that reason

Reply:
@keithf4 yeah, I'd noticed some in the replacement text (i.e.,: ``` `crunchy-prometheus.yml`  ```)

Reply:
Sorry, missed the ones in the description section. Fixed.